source/index.js

```diff
@@ -139,7 +139,7 @@ module.exports = async (input = 'patch', options) => {
 				title: 'Installing dependencies using npm',
 				enabled: () => options.yarn === false,
 				task: () => {
-					const args = hasLockFile ? ['ci'] : ['install', '--no-package-lock', '--no-production'];
+					const args = hasLockFile ? ['ci'] : ['install', '--no-package-lock', '--no-production', '--engine-strict'];
```

Comment:
Shouldn't the flag be specified when running `$ npm ci` as well?

Reply:
I wrote a whole paragraph about that in the PR description.

https://github.com/sindresorhus/np/pull/481#issue-356704421

Reply:
Sorry, I completely missed that part. This seems like a [known bug](https://npm.community/t/npm-ci-ignores-engine-strict/10447), and so let’s add it anyways (like Sindre wrote above).